docker-compose.yml

```diff
@@ -40,23 +8,8 @@ services:
       dockerfile: ./web/Dockerfile.web
       args:
         DOCKER_BUILDKIT: 1
-        NEXT_PUBLIC_API_BASE_URL: http://localhost:8000
-        NEXT_PUBLIC_DEPLOY_URL: http://localhost/spaces
     restart: always
     command: /usr/local/bin/start.sh web/server.js web
-    env_file:
```

Comment:
Are we not passing any environment variables here @henit-chobisa

Reply:
Not Nikhil, you see here we are directly passing the .env file from the apiserver, we don't need to pass any environment variables.